homa_experiments/homa_test_w1.config

```diff
@@ -0,0 +1,204 @@
+tests {
+  attributes: { key: 'workload' value: 'w1' }
+  attributes: { key: 'max_message_length' value: '1000000' }
+  attributes: { key: 'min_bucket_frac' value: '0.0025' }
+  attributes: { key: 'max_size_ratio' value: '1.2' }
+  attributes: { key: 'Gbps' value: '1' }
+  attributes: { key: 'num_nodes' value: '10' }
+  attributes: { key: 'max_duration' value: '20000' }
```

Comment:
This says to run the traffic for 0.02 seconds. We probably want more like 10 seconds